internal/server/recon/golden/bulk_find_entities/result.json

```diff
@@ -26,10 +26,7 @@
       ]
     },
     {
-      "description": "mmountan view",
-      "dcids": [
-        "geoId/0649670"
-      ]
+      "description": "mmountan view"
```

Comment:
this test is flaky due to maps api, maybe we should remove it @Spaceenter ?

Reply:
Right, Maps Place API is flaky... Do we completely remove the golden test for (bulk)findEntities?
Also add @pradh for visibility.

Reply:
Sure, let's remove it for now to unblock.

Reply:
done - removed